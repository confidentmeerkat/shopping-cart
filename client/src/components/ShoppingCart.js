import React from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Header from './Header';
import Account from './Account';
import Cart from './Cart';
import Homepage from './Homepage';
import ProductDetails from './ProductDetails';
import '../styles/ShoppingCart.css';

const ShoppingCart = () => (
  <BrowserRouter>
    <MuiThemeProvider>
      <div className="container">
        <Header />
        <Route exact path="/" component={Homepage} />
        <Route path="/account" component={Account} />
        <Route path="/cart" component={Cart} />
        <Route path="/product/:id" component={ProductDetails} />
      </div>
    </MuiThemeProvider>
  </BrowserRouter>
);

export default ShoppingCart;