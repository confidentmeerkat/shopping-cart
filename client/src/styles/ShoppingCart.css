button:focus {
  outline: 0;
}

body {
  margin: 0;
}

.container {
  width: 100%;
  max-width: 1200px;
  margin: auto;
  font-family: 'Roboto';
  color: #2e3a4edc;
}