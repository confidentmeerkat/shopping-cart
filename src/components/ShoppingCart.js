import React from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Header from './Header';
import FiltersList from './FiltersList';
import Products from './Products';
import '../styles/ShoppingCart.css';

const ShoppingCart = () => {
  return (
    <div>
      <BrowserRouter>
        <div className="container">
          <MuiThemeProvider>
            <Header/>
              <div className="main">
                <FiltersList/>
                <Products/>
              </div>
          </MuiThemeProvider>
        </div>
      </BrowserRouter>
    </div>
  )
};

export default ShoppingCart;