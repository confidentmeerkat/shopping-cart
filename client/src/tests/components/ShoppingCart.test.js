import React from 'react';
import { shallow } from 'enzyme';
import ShoppingCart from '../../components/ShoppingCart';

describe('<ShoppingCart />', () => {
  it('renders ShoppingCart correctly', () => {
    const wrapper = shallow(<ShoppingCart />);

    expect(wrapper).toMatchSnapshot();
  });
});