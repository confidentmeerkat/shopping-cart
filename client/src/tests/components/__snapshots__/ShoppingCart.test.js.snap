// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShoppingCart /> renders ShoppingCart correctly 1`] = `
<BrowserRouter>
  <MuiThemeProvider>
    <div
      className="container"
    >
      <Route
        component={[Function]}
      />
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          path="/account"
        />
        <Route
          component={[Function]}
          path="/cart"
        />
        <Route
          component={[Function]}
          path="/product/:id"
        />
        <Route
          component={[Function]}
        />
      </Switch>
      <Route
        component={[Function]}
      />
    </div>
  </MuiThemeProvider>
</BrowserRouter>
`;
